src/roman_datamodels/datamodels/_core.py

```diff
@@ -137,9 +137,9 @@ def __del__(self):
         """Ensure closure of resources when deleted."""
         self.close()
 
-    def copy(self, memo=None):
+    def copy(self, deepcopy=True, memo=None):
         result = self.__class__(init=None)
-        self.clone(result, self, deepcopy=True, memo=memo)
+        self.clone(result, self, deepcopy=deepcopy, memo=memo)
```

Comment:
Note that I think this `clone` vs `copy` thing and the `clone` interface are both bizarre. In reality it should just all be unified into `copy`. Doing that is outside the scope of the reducing copies PR, so it can wait.